bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java

```diff
@@ -347,20 +348,53 @@ public boolean isChannelLinked(ChannelUID channelUID) {
         }
     };
 
-    private ThingRegistryImpl thingRegistry;
-
-    private BundleResolver bundleResolver;
-
-    private ConfigDescriptionRegistry configDescriptionRegistry;
-    private ConfigDescriptionValidator configDescriptionValidator;
+    @Activate
+    public ThingManagerImpl( //
+            final @Reference BundleResolver bundleResolver,
+            final @Reference ChannelGroupTypeRegistry channelGroupTypeRegistry,
+            final @Reference ChannelTypeRegistry channelTypeRegistry,
+            final @Reference CommunicationManager communicationManager,
+            final @Reference ConfigDescriptionRegistry configDescriptionRegistry,
+            final @Reference ConfigDescriptionValidator configDescriptionValidator,
+            final @Reference EventPublisher eventPublisher,
+            final @Reference ItemChannelLinkRegistry itemChannelLinkRegistry,
+            final @Reference ReadyService readyService, //
+            final @Reference SafeCaller safeCaller, //
+            final @Reference StorageService storageService, //
```

Comment:
I always use this trick to force the auto formatter to start the next item on a new line. :wink:
That way you don't have to use formatter:on/off for this particular case.